viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1065,6 +1074,37 @@ export class Cognite3DViewer {
     return intersections.length > 0 ? intersections[0] : null;
   }
 
+  /**
+   * Raycasting model(s) for finding where the ray intersects with the model. Note: Only Cad models.
+   * @param offsetX X coordinate in pixels (relative to the domElement).
+   * @param offsetY Y coordinate in pixels (relative to the domElement).
+   * @returns A promise that if there was an intersection then return the intersection point.
+   */
+  async getIntersectionPixelFromBuffer(offsetX: number, offsetY: number): Promise<THREE.Vector3> {
+    const cadModels = this.getModels('cad');
+    const cadNodes = cadModels.map(x => x.cadNode);
+    const normalizedCoords = {
+      x: (offsetX / this.renderer.domElement.clientWidth) * 2 - 1,
+      y: (offsetY / this.renderer.domElement.clientHeight) * -2 + 1
+    };
+
+    const input: IntersectInput = {
+      normalizedCoords,
+      camera: this.camera,
+      renderer: this.renderer,
+      clippingPlanes: this.getClippingPlanes(),
+      domElement: this.renderer.domElement
+    };
+
+    //Retrive the full pixel buffer if the camera has changed
+    let cameraChanged = false;
+    this.cameraManager.on('cameraChange', function () {
+      cameraChanged = true;
+    });
+
+    return intersectionCadNodesFromStoredPixel(cadNodes, input, cameraChanged);
```

Comment:
This makes no sense. cameraChanged is always false inside `intersectionCadNodesFromStoredPixel`